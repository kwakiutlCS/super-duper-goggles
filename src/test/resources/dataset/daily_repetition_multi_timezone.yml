reminder:
- id: 1
  content: westeurope
  user_id: user
  created_at: 1000
  updated_at: 1000
  time_id: 2
- id: 3
  content: westeuropesummer
  user_id: user
  created_at: 1000
  updated_at: 1000
  time_id: 4
- id: 5
  content: centraleurope
  user_id: user
  created_at: 1000
  updated_at: 1000
  time_id: 6
- id: 7
  content: centraleuropesummer
  user_id: user
  created_at: 1000
  updated_at: 1000
  time_id: 8
time:
- id: 2
  timestamp: 1609504200
  seconds_in_day: 45000
  unit: DAYS
  step: 1
  bound_type: NO_BOUND
  zone: "Europe/Lisbon"
- id: 4
  timestamp: 1625139000
  seconds_in_day: 45000
  unit: DAYS
  step: 1
  bound_type: NO_BOUND
  zone: "Europe/Lisbon"
- id: 6
  timestamp: 1609500600
  seconds_in_day: 41400
  unit: DAYS
  step: 1
  bound_type: NO_BOUND
  zone: "Europe/Berlin"
- id: 8
  timestamp: 1625135400
  seconds_in_day: 41400
  unit: DAYS
  step: 1
  bound_type: NO_BOUND
  zone: "Europe/Berlin"