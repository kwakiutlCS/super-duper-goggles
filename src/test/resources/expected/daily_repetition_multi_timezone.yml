reminder:
- id: "regex:\\d+"
  content: westeurope
  user_id: user
  created_at: 1000
  updated_at: 1000
- id: "regex:\\d+"
  content: westeuropesummer
  user_id: user
  created_at: 1000
  updated_at: 1000
- id: "regex:\\d+"
  content: centraleurope
  user_id: user
  created_at: 1000
  updated_at: 1000
- id: "regex:\\d+"
  content: centraleuropesummer
  user_id: user
  created_at: 1000
  updated_at: 1000
time:
- id: "regex:\\d+"
  timestamp: 1609504200
  seconds_in_day: 45000
  unit: DAYS
  step: 1
  bound_type: NO_BOUND
  zone: "Europe/Lisbon"
- id: "regex:\\d+"
  timestamp: 1625139000
  seconds_in_day: 45000
  unit: DAYS
  step: 1
  bound_type: NO_BOUND
  zone: "Europe/Lisbon"
- id: "regex:\\d+"
  timestamp: 1609500600
  seconds_in_day: 41400
  unit: DAYS
  step: 1
  bound_type: NO_BOUND
  zone: "Europe/Berlin"
- id: "regex:\\d+"
  timestamp: 1625135400
  seconds_in_day: 41400
  unit: DAYS
  step: 1
  bound_type: NO_BOUND
  zone: "Europe/Berlin"